tortoise/ninja_tortoise.go

```diff
@@ -444,7 +443,11 @@ func (ni *ninjaTortoise) addPatternVote(p votingPattern, view map[types.BlockID]
 		for _, ex := range vp {
 			blocks, err := ni.db.LayerBlockIds(ex.Layer())
 			if err != nil {
-				ni.logger.Panic("could not retrieve layer block ids")
+				if ex.Layer() == 0 {
```

Comment:
wouldn't it be better to check this before trying to get the blockids from the db?

Reply:
It should only happen once and the warning is simply a patch until we have new tortoise, I'd rather keep it here because logically the warning should come only if we didn't read blocks from layer 0